source/routing/dataraptor.cpp

```diff
@@ -67,28 +69,29 @@ void dataRAPTOR::JppsFromSp::filter_jpps(const boost::dynamic_bitset<>& valid_jp
     }
 }
 
-void dataRAPTOR::JppsFromJp::load(const type::PT_Data &data) {
-    jpps_from_jp.assign(data.journey_patterns);
-    for (const auto* jp: data.journey_patterns) {
-        const bool has_freq = !jp->frequency_vehicle_journey_list.empty();
-        for (const auto* jpp: jp->journey_pattern_point_list) {
-            jpps_from_jp[JpIdx(*jp)].push_back(
-                {JppIdx(*jpp), SpIdx(*jpp->stop_point), has_freq});
+void dataRAPTOR::JppsFromJp::load(const JourneyPatternContainer& jp_container) {
+    jpps_from_jp.assign(jp_container.get_jps_values());
```

Comment:
hmmm, IMO, this line quite confusing.... we're assigning a vector of jps to a vector named jpps....